packages/synapse-react-client/src/theme/DefaultTheme.tsx

```diff
@@ -426,6 +426,13 @@ export const defaultMuiThemeOptions: ThemeOptions = {
         },
       },
     },
+    MuiToggleButton: {
+      styleOverrides: {
+        root: {
+          textTransform: 'var(--synapse-button-text-transform)' as any,
+        },
+      },
+    },
```

Comment:
For `MuiToggleButton` (first time we are using it), add styleOverride for `text-transform`, which is also in `MuiButton`